x-pack/plugins/siem/server/graphql/kpi_hosts/schema.gql.ts

```diff
@@ -7,15 +7,16 @@
 import gql from 'graphql-tag';
 
 export const kpiHostsSchema = gql`
-  type Count {
+  type KpiHostCount {
     value: Float
     doc_count: Float
   }
 
   type KpiHostHistogramData {
     key: Float
     key_as_string: String
-    count: Count
+    count: KpiHostCount
+    doc_count: Float
```

Comment:
Question: what is the diff between this doc_count and the one inside of KpiHostCount